README.md

```diff
@@ -30,18 +30,31 @@ There is currently one public instance running at
 
 # Development / Running
 
+## Dependencies
+
+If you've yet to install `bower` and `gulp`, run
+
+``` $ npm install -g bower ```
+
+``` $ npm install -g gulp ```
+
+
+## Install and Run
+
 To install and build static assets:
 
 ``` $ npm install ```
 
+``` $ mkdir build ```
+
+``` $ cd static/ && bower install && cd .. ```
```

Comment:
Good catch. Eventually I'd like to move everything into npm tasks (that refer to node_modules/.bin/gulp, etc, so global binaries aren't needed.